DeployAzureVM.ps1

```diff
@@ -65,12 +64,20 @@ if ($RGdeployment.ProvisioningState -eq "Succeeded")
 }
 
 #To get the PublicIp Address of the VM that created
-Write-Host "Getting the IpAddress of the VM"
-if ($RGdeployment.ProvisioningState -eq "Succeeded")
+Function GetIPAddress
 {
-    Get-AzureRmPublicIpAddress -ResourceGroupName $resourceGroupName  -Name MyPublicIp | Select-Object ResourceGroupName, Name, IpAddress
-}
-else
-{
-    Write-Host "IpAddress not found"
+Write-Host "getting the adminUsername and IPAddress"
+$var = Get-Content C:\Users\v-bhvenn\Desktop\Azurevmdeploy\azuredeploy.parameters.json | ConvertFrom-Json
```

Comment:
Please use "Templates\azuredeploy.parameters.json" for params  file dont use any file.